client/src/index.js

```diff
@@ -223,6 +231,8 @@ class MainView extends React.Component {
   }
 }
 
+export default connect(mapStateToProps)(MainView)
```

Comment:
You don't want to export anything from the main file. `connect()` returns a Redux-connected component, so instead use:

```js
const ConnectedMainView = connect(mapStateToProps)(MainView)
```

And use `<ConnectedMainView>` instead of `<MainView>`. Should probably refactor the `MainView` out of `index.js` but that can be done separetely (maybe with general index/site/proxy entrypoint merge and cleanup PR)